### ISSUE TITLE
Hi, I Have One Problem

### ISSUE THREAD

**Issue by adikom** [martin-ger/uMQTTBroker#22]
Hi, I have Wemos D1 Mini,

I can subscribe from mosquitto but when i publish anything can just 3 time. After mosquitto says Connection was lost.

**Comment by adikom** [comment]
And can i use my WeMos, Broker and Client both.

**Comment by martin-ger** [comment]
Yes, the ESP can be both at the same time - broker and client to another broker. For the first question I do not exactly understand the setup.

**Comment by adikom** [comment]
Hi, thanks for your help/answer.

I'm using my WeMos as broker, another device can connect it.
But another device can publish just 3 times, after device disconnecting.

**Comment by martin-ger** [comment]
Don't think that this is a known problem. Do you re-connect three times or do you simple publish three times over one connection?

**Comment by adikom** [comment]
I think mosquitto connect per one publishing

**Comment by adikom** [comment]
![adsiz](https://user-images.githubusercontent.com/40799781/46671843-889d7980-cbde-11e8-93cc-76909da22888.png)

**Comment by adikom** [comment]
just four time... :(

**Comment by martin-ger** [comment]
Will look into that later today.

**Comment by adikom** [comment]
I tried it and yes. When I reconnect 4 times, after I can't publish anything.

**Comment by adikom** [comment]
Fix:
Important: Use the setting "lwip Variant: 1.4 High Bandwidth" in the "Tools" menu

Sorry... thanks for your helps.
